City-of-Helsinki/drupal-helfi-platform-config
# UHF-2059: Admin UI translations

# Admin UI translations [UHF-2059](https://helsinkisolutionoffice.atlassian.net/browse/UHF-2059)
Alter Admin UI translations based on customer needs.

## What was done
* Added patch for paragraph widget to remove duplicate information of where the paragraph is being added
* Removed `- Not specified -` and `- Not applicable -` values from translations when creating new content
* Refactored titles and descriptions of most generic fields found in [this excel](https://helsinginkaupunki-my.sharepoint.com/:x:/g/personal/aleksi_salonen_hel_fi/EVGs4asmBxdIpGrQrjVcYH0BaLSzARPWyH-unZP68PtTpg) 

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-2059_admin_ui_translations && composer require drupal/hdbt_admin:dev-UHF-2059_admin_ui_translations`
* Run `make drush-updb`
* Run `make shell`
  * In shell, run `drush locale:check && drush locale:update && drush helfi:locale-import helfi_platform_config && drush cr`

## How to test
* Compare [the excel](https://helsinginkaupunki-my.sharepoint.com/:x:/g/personal/aleksi_salonen_hel_fi/EVGs4asmBxdIpGrQrjVcYH0BaLSzARPWyH-unZP68PtTpg) with actual content type and field titles/descriptions
* Check English and Finnish translations

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/108
